bitnami/dokuwiki/templates/deployment.yaml

```diff
@@ -48,6 +48,25 @@ spec:
           hostnames:
             - "status.localhost"
       initContainers:
+        {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+        - name: volume-permissions
+          image: {{ include "dokuwiki.volumePermissions.image" . }}
+          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              mkdir -p "/bitnami/dokuwiki"
+              chown -R "{{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}" "/bitnami/dokuwiki"
```

Comment:
Maybe we want to use a "default" in case `containerSecurityContext.runAsUser` or `podSecurityContext.fsGroup` are not set

Reply:
Are not "defaults" set in `values.yaml`?

```yaml
## SecurityContext configuration for the pods
##
podSecurityContext:
  enabled: true
  fsGroup: 1001

## SecurityContext configuration for the container
##
containerSecurityContext:
  enabled: true
  runAsUser: 1001
```

Reply:
I guess that's enough